pandas/tests/tseries/test_frequencies.py

```diff
@@ -812,6 +826,7 @@ def test_legacy_offset_warnings(self):
                  'W@FRI', 'W@SAT', 'W@SUN', 'Q@JAN', 'Q@FEB', 'Q@MAR',
                  'A@JAN', 'A@FEB', 'A@MAR', 'A@APR', 'A@MAY', 'A@JUN',
                  'A@JUL', 'A@AUG', 'A@SEP', 'A@OCT', 'A@NOV', 'A@DEC',
+                 'Y@JAN',
```

Comment:
Nit : can we shift components left (and up) so that we have some on the same line as this one (and perhaps one orphan one at the end)?